en/docs/architecture-diagram/architecture-diagram.md

```diff
@@ -1,24 +1,17 @@
-# Project Design Diagram
+# Project Design view
 
-With the increasing size and complexity, software applications have evolved to be a composition of multiple interrelated components. When the number and/or depth of such components grow, it becomes convenient to have a high-level picture of the system components and their interactions with one another. The Ballerina Project Design View is deployed with the intention of providing a convenient way to generate such a high-level picture of projects purely based on the source code. In addition to the component interactions, the Project Design View is also capable of visualizing the Ballerina record types in the project workspace and their associations.
+Software applications have evolved into a composition of multiple interrelated components to be able to handle increasing complexities. When the number and/or depth of such components grow, it is convenient to have a high-level picture of the system components and their interactions with one another. 
 
```

Comment:
Prerequisites section was removed as Installing and setting up is not discussed in all other pages. Therefore this looked a bit redundant, which is why it was removed.